packages/calcite-components/src/components/carousel/carousel.tsx

```diff
@@ -38,14 +38,17 @@ import {
   T9nComponent,
   updateMessages,
 } from "../../utils/t9n";
+import { createObserver } from "../../utils/observers";
+import { breakpoints } from "../../utils/responsive";
 import { getRoundRobinIndex } from "../../utils/array";
-import { CSS, DURATION, ICONS } from "./resources";
+import { CSS, DURATION, ICONS, maxItemBreakpoints } from "./resources";
 import { CarouselMessages } from "./assets/carousel/t9n";
 import { ArrowType, AutoplayType } from "./interfaces";
 
 /**
  * @slot - A slot for adding `calcite-carousel-item`s.
  */
+
```

Comment:
For consistency, can you remove the newline between the JSDoc block and the component?